README.md

```diff
@@ -179,6 +179,80 @@ If the request was successful, you will need to store the authy id in your datab
   Hash sms = users.requestSms(authy_id, options);
 ```
 
+## OneTouch Support
+
+  `client.getOneTouch()` will return the OneTouch wrapper, with this component you will be able to use all the 
+  endpoints defined in the [OneTouch API](https://www.twilio.com/docs/api/authy/authy-onetouch-api), 
+  for example, using the 
+  'sendApprovalRequest' method you will create an ApprovalRequest
+  in OneTouch, then the approval request
+  status can be queried by using the  `uuid` provided. (NOTE: The UUID is the unique identifier for the request sent to OneTouch)    
+
+```java
+    HashMap<String, String> details = new HashMap<String, String>();
+    details.put("username", "Bill Smith");
+    details.put("location", "California, USA");
+    details.put("amount", "$20.000");
+
+    HashMap<String, String> hidden = new HashMap<String, String>();
+    hidden.put("ip_address", "10.10.3.203");
+
+    HashMap<String, String> logos = new HashMap<String, String>();
+
+    OneTouchResponse response = client.getOneTouch().sendApprovalRequest(Integer.parseInt(103), "Authorize OneTouch Unit Test", details, hidden, logos, 86400);
```

Comment:
@hans-maws lets wait before you make changes here base on suggestion. I'll review and make a further comment.